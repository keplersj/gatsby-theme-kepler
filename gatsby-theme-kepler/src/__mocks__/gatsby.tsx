import * as React from "react";
const gatsby = jest.requireActual("gatsby");

module.exports = {
  ...gatsby,
  graphql: jest.fn(),
  Link: jest.fn(
    ({
      // these props are invalid for an `a` tag
      /* eslint-disable @typescript-eslint/no-unused-vars */
      /* eslint-disable no-unused-vars */
      activeClassName,
      activeStyle,
      getProps,
      innerRef,
      partiallyActive,
      ref,
      replace,
      /* eslint-enable @typescript-eslint/no-unused-vars */
      /* eslint-enable no-unused-vars */
      to,
      ...rest
      // eslint-disable-next-line jsx-a11y/anchor-has-content
    }): React.ReactElement => <a {...rest} href={to} />
  ),
  StaticQuery: jest.fn(),
  useStaticQuery: jest.fn()
};
