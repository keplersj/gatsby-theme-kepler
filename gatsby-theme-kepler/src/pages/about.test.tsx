import * as React from "react";
import renderer from "react-test-renderer";
import { useStaticQuery } from "gatsby";
import Page from "./about";
import {
  KeplerAvatarComponentQuery,
  KeplerBannerBackgroundData,
  KeplerNavbarData,
  KeplerBaseLayoutData,
  KeplerSocialLinksQuery
} from "../__mockData__";
import deepMerge from "deepmerge";

beforeEach((): void => {
  (useStaticQuery as jest.Mock).mockImplementation((): object =>
    deepMerge.all([
      KeplerBannerBackgroundData,
      KeplerAvatarComponentQuery,
      KeplerNavbarData,
      KeplerBaseLayoutData,
      KeplerSocialLinksQuery
    ])
  );
});

describe("About Page", (): void => {
  it("renders correctly", (): void => {
    const tree = renderer
      .create(
        <Page
          location={{ pathname: "/about/" }}
          data={{
            biography: {
              childMdx: {
                frontmatter: {
                  title: "Biography"
                },
                body: ""
              }
            },
            experience: {
              edges: [
                {
                  node: {
                    id: "example-experience-listing",
                    childMdx: {
                      id: "example-experience-listing-mdx",
                      body: "",
                      frontmatter: {
                        title: "Example Workplace",
                        position: "Sample Position",
                        /* eslint-disable @typescript-eslint/camelcase */
                        rawEnd: "2018-12-18",
                        end_date: "Dec 2018",
                        rawStart: "2017-12-18",
                        start_date: "Dec 2017"
                        /* eslint-enable @typescript-eslint/camelcase */
                      }
                    }
                  }
                }
              ]
            },
            education: {
              edges: [
                {
                  node: {
                    id: "example-education-listing",
                    childMdx: {
                      id: "example-education-listing-mdx",
                      body: "",
                      frontmatter: {
                        title: "Example Institution",
                        degree: "Sample Degree",
                        /* eslint-disable @typescript-eslint/camelcase */
                        rawEnd: "2018-12-18",
                        end_date: "Dec 2018",
                        rawStart: "2017-12-18",
                        start_date: "Dec 2017"
                        /* eslint-enable @typescript-eslint/camelcase */
                      }
                    }
                  }
                }
              ]
            },
            skills: {
              childMdx: {
                frontmatter: {
                  title: "Skills"
                },
                body: ""
              }
            }
          }}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
