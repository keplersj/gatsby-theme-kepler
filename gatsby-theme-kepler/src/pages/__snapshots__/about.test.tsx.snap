// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page renders correctly 1`] = `
Array [
  <head>
    Object {}
    Object {
      "lang": "en",
    }
    <link
      data-react-helmet={true}
      href="/about/"
      rel="canonical"
    />
    <meta
      content=""
      data-react-helmet={true}
      name="description"
    />
    <meta
      content="Kepler Sticka-Jones"
      data-react-helmet={true}
      property="og:site_name"
    />
    <meta
      content="Kepler Sticka-Jones"
      data-react-helmet={true}
      property="og:title"
    />
    <meta
      content="/about/"
      data-react-helmet={true}
      property="og:url"
    />
    <meta
      content=""
      data-react-helmet={true}
      property="og:description"
    />
    <meta
      content="summary"
      data-react-helmet={true}
      name="twitter:card"
    />
    <meta
      content="@test"
      data-react-helmet={true}
      name="twitter:site"
    />
    <meta
      content="@test"
      data-react-helmet={true}
      name="twitter:creator"
    />
    <script
      data-react-helmet={true}
      type="application/ld+json"
    >
      Object {
      "@context": "https://schema.org/",
      "@type": "BreadcrumbList",
      "itemListElement": Array [
        Object {
          "@type": "ListItem",
          "item": Object {
            "@id": "/",
            "@type": "WebPage",
            "name": "Kepler Sticka-Jones",
          },
          "position": 1,
        },
        Object {
          "@type": "ListItem",
          "item": Object {
            "@id": "/about/",
            "@type": "WebPage",
            "name": "About",
          },
          "position": 2,
        },
      ],
    }
    </script>
    <title
      data-react-helmet={true}
    >
      Kepler Sticka-Jones
    </title>
  </head>

<a
    data-reach-skip-link={true}
    href="#reach-skip-nav"
  >
    Skip to content
  </a>,
  .emotion-8 {
  position: relative;
}

.emotion-8::before,
.emotion-8::after {
  content: "";
  display: block;
  position: absolute;
  width: 100%;
  height: 100%;
  top: 0;
  left: 0;
  -webkit-transition: opacity 0.5s ease 0.25s;
  transition: opacity 0.5s ease 0.25s;
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
}

.emotion-8::after {
  z-index: -100;
  opacity: 0;
  background-image: url("/static/banner_example/1/banner.jpg");
}

@media screen and (prefers-color-scheme:dark) {
  .emotion-8::after {
    background-image: url("/static/banner_example/1/banner_dark.jpg");
  }
}

.emotion-8::before {
  z-index: -101;
  opacity: 1;
  background-image: url("data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAPABQDASIAAhEBAxEB/8QAGAAAAgMAAAAAAAAAAAAAAAAAAAYBAgX/xAAVAQEBAAAAAAAAAAAAAAAAAAAAAf/aAAwDAQACEAMQAAAB1ZUrU0CyL//EABoQAAICAwAAAAAAAAAAAAAAAAACARIREyH/2gAIAQEAAQUCiFMIUg6XY2Mf/8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAwEBPwE//8QAFhEBAQEAAAAAAAAAAAAAAAAAAAER/9oACAECAQE/AW1//8QAGBAAAwEBAAAAAAAAAAAAAAAAAAIyEDH/2gAIAQEABj8C3pbFsf/EAB4QAAEDBAMAAAAAAAAAAAAAAAABEUEhUWFxkdHx/9oACAEBAAE/IcnggdtoLfKJjfce4f/aAAwDAQACAAMAAAAQGw//xAAWEQEBAQAAAAAAAAAAAAAAAAABABH/2gAIAQMBAT8QAsL/xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAgEBPxBWH//EABwQAQACAgMBAAAAAAAAAAAAAAEAESFRcbHRwf/aAAgBAQABPxBpi3Fuos5FpCIbpBzBcLAlYNX9gZ9/s//Z");
}

@media screen and (prefers-color-scheme:dark) {
  .emotion-8::before {
    background-image: url("data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAPABQDASIAAhEBAxEB/8QAGAAAAgMAAAAAAAAAAAAAAAAAAAYBAgX/xAAVAQEBAAAAAAAAAAAAAAAAAAAAAf/aAAwDAQACEAMQAAAB1ZUrU0CyL//EABoQAAICAwAAAAAAAAAAAAAAAAACARIREyH/2gAIAQEAAQUCiFMIUg6XY2Mf/8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAwEBPwE//8QAFhEBAQEAAAAAAAAAAAAAAAAAAAER/9oACAECAQE/AW1//8QAGBAAAwEBAAAAAAAAAAAAAAAAAAIyEDH/2gAIAQEABj8C3pbFsf/EAB4QAAEDBAMAAAAAAAAAAAAAAAABEUEhUWFxkdHx/9oACAEBAAE/IcnggdtoLfKJjfce4f/aAAwDAQACAAMAAAAQGw//xAAWEQEBAQAAAAAAAAAAAAAAAAABABH/2gAIAQMBAT8QAsL/xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAgEBPxBWH//EABwQAQACAgMBAAAAAAAAAAAAAAEAESFRcbHRwf/aAAgBAQABPxBpi3Fuos5FpCIbpBzBcLAlYNX9gZ9/s//Z");
  }
}

.emotion-0 {
  opacity: 1;
  background-image: url("/static/banner_example/1/banner.jpg");
}

@media screen and (prefers-color-scheme:dark) {
  .emotion-0 {
    background-image: url("/static/banner_example/1/banner_dark.jpg");
  }
}

.emotion-7 {
  height: 3rem;
  max-width: 100vw;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-backdrop-filter: blur(5px);
  backdrop-filter: blur(5px);
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding-left: 1rem;
  padding-right: 1rem;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.emotion-2 {
  text-align: start;
  max-width: 50%;
}

.emotion-1 {
  font-weight: 600;
  color: #141414;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-1:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

@media (prefers-color-scheme:dark) {
  .emotion-1 {
    color: white;
  }
}

.emotion-6 {
  text-align: end;
  max-width: 50%;
}

.emotion-6 > :not(:last-child) {
  margin-right: 1em;
}

<div
    className="emotion-8"
  >
    <noscript
      className="emotion-0"
    />
    <div
      className="emotion-7"
    >
      <div
        className="emotion-2"
      >
        <a
          className="emotion-1"
          href="/"
        >
          Kepler Sticka-Jones
        </a>
      </div>
      <div
        className="emotion-6"
      >
        <a
          className="emotion-1"
          href="/blog"
        >
          Blog
        </a>
        <a
          className="emotion-1"
          href="/projects"
        >
          Projects
        </a>
        <a
          className="emotion-1"
          href="/about"
        >
          About
        </a>
      </div>
    </div>
  </div>,
  <div
    id="reach-skip-nav"
  />,
  .emotion-14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 1em;
}

@media (max-width:839px) {
  .emotion-14 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

.emotion-8 {
  margin: 1em;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-width: 33vw;
}

@media (max-width:839px) {
  .emotion-8 {
    min-width: inherit;
  }
}

.emotion-0 {
  margin-bottom: 10px;
  border-radius: 140px;
}

.emotion-1 {
  font-size: 2em;
}

.emotion-2 {
  font-size: 1em;
}

.emotion-7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.emotion-3 {
  padding: 0.5em;
}

.emotion-13 {
  margin: 1em;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-9:not(:last-of-type)::after {
  content: " Â· ";
}

<div
    className="emotion-14"
    role="region"
  >
    <header
      className="emotion-8"
    >
      <div
        aria-label="Avatar"
        role="img"
      >
        <div
          className="emotion-0 gatsby-image-wrapper"
          style={
            Object {
              "display": "inline-block",
              "height": 200,
              "overflow": "hidden",
              "position": "relative",
              "width": 200,
            }
          }
        >
          <picture>
            <source
              srcSet="/static/example/2/avatar.jpg 1x,
                /static/example/3/avatar.jpg 1.5x,
                /static/example/4/avatar.jpg 2x"
            />
            <img
              alt="Avatar"
              height={200}
              loading="lazy"
              onLoad={[Function]}
              src="/static/example/1/avatar.jpg"
              srcSet="/static/example/2/avatar.jpg 1x,
                /static/example/3/avatar.jpg 1.5x,
                /static/example/4/avatar.jpg 2x"
              style={
                Object {
                  "height": "100%",
                  "left": 0,
                  "objectFit": "cover",
                  "objectPosition": "center",
                  "opacity": 0,
                  "position": "absolute",
                  "top": 0,
                  "transition": "opacity 500ms",
                  "width": "100%",
                }
              }
              width={200}
            />
          </picture>
          <noscript
            dangerouslySetInnerHTML={
              Object {
                "__html": "<picture><source srcset=\\"/static/example/2/avatar.jpg 1x,
                /static/example/3/avatar.jpg 1.5x,
                /static/example/4/avatar.jpg 2x\\" /><img loading=\\"lazy\\" width=\\"200\\" height=\\"200\\" srcset=\\"/static/example/2/avatar.jpg 1x,
                /static/example/3/avatar.jpg 1.5x,
                /static/example/4/avatar.jpg 2x\\" src=\\"/static/example/1/avatar.jpg\\" alt=\\"Avatar\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
              }
            }
          />
        </div>
      </div>
      <span
        aria-level={1}
        className="emotion-1"
        role="heading"
        title="Name"
      >
        Kepler Sticka-Jones
      </span>
      <span
        aria-level={2}
        className="emotion-2"
        role="heading"
        title="Location"
      >
        Salt Lake City, UT, USA
      </span>
      <address
        className="emotion-7"
        id="contact"
        role="list"
      >
        <a
          className="emotion-3"
          href="https://twitter.com/test"
          id="@test"
          name="Twitter"
          rel="noopener noreferrer"
          role="listitem"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1.5em"
            size="1.5em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1.5em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              Twitter: @test
            </title>
            <path
              d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
            />
          </svg>
        </a>
        <a
          className="emotion-3"
          href="https://github.com/test"
          id="test"
          name="GitHub"
          rel="noopener noreferrer"
          role="listitem"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1.5em"
            size="1.5em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 496 512"
            width="1.5em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              GitHub: test
            </title>
            <path
              d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
            />
          </svg>
        </a>
        <a
          className="emotion-3"
          href="https://twitter.com/test"
          id="@test"
          name="Twitter"
          rel="noopener noreferrer"
          role="listitem"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1.5em"
            size="1.5em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1.5em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              Twitter: @test
            </title>
            <path
              d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
            />
          </svg>
        </a>
        <a
          className="emotion-3"
          href="https://github.com/test"
          id="test"
          name="GitHub"
          rel="noopener noreferrer"
          role="listitem"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1.5em"
            size="1.5em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 496 512"
            width="1.5em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              GitHub: test
            </title>
            <path
              d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
            />
          </svg>
        </a>
      </address>
    </header>
    <main
      className="emotion-13"
    >
      <div>
        <h1>
          Biography
        </h1>
        <p>
          
        </p>
      </div>
      <div>
        <h1>
          Experience
        </h1>
        <article>
          <h2>
            Sample Position
          </h2>
          <span
            className="emotion-9"
          >
            Example Workplace
          </span>
          <span
            className="emotion-9"
          >
            <time
              dateTime="2017-12-18"
            >
              Dec 2017
            </time>
             - 
            <time
              dateTime="2018-12-18"
            >
              Dec 2018
            </time>
          </span>
          <p>
            
          </p>
        </article>
      </div>
      <div>
        <h1>
          Education
        </h1>
        <article>
          <h2>
            Example Institution
          </h2>
          <span
            className="emotion-9"
          >
            Sample Degree
          </span>
          <span
            className="emotion-9"
          >
            <time
              dateTime="2017-12-18"
            >
              Dec 2017
            </time>
             - 
            <time
              dateTime="2018-12-18"
            >
              Dec 2018
            </time>
          </span>
          <p>
            
          </p>
        </article>
      </div>
      <div>
        <h1>
          Skills
        </h1>
        <p>
          
        </p>
      </div>
    </main>
  </div>,
]
`;
